src/test/client/featureToggleService.ts

```diff
@@ -0,0 +1,93 @@
+import { FeatureToggleService } from './../../main/client/featureToggleService'
+import * as FeatureToggleApiMock from './../http-mocks/featureToggleApiMock'
```

Comment:
Please rename to be `featureToggleApiMock`.

Reply:
Do you want the name FeatureToggleApiMock changed to featureToggleApiMock? The names begin in upper case in all other classes.

Reply:
Yes. It is similar to `import * as requestPromise from 'request-promise-native'` due to the fact not class but functions are exported. I expect to have more `import * as featureToggleApiMock`